Sdk/MoyasarSdk/Models/CreditCardViewModel.swift

```diff
@@ -75,9 +75,10 @@ public class CreditCardViewModel: ObservableObject {
     lazy var expiryValidator = ExpiryValidator()
     lazy var securityCodeValidator = SecurityCodeValidator(getNumber: { self.number })
     
-    public init(paymentRequest: PaymentRequest, resultCallback: @escaping ResultCallback) {
+    public init(apiKey: String, paymentRequest: PaymentRequest, resultCallback: @escaping ResultCallback) {
```

Comment:
@AbdulazizAlrabiah is there a reason here for taking the `apiKey` separately? Can we instead just add it within the `PaymentRequest` object?

Reply:
Same goes for the `baseUrl`

Reply:
Right, it's better that way

Just submitted required changes